src/components/Editor/EditorMenu.js

```diff
@@ -78,7 +79,10 @@ function getMenuItems(
     label: revealInTreeLabel,
     accesskey: revealInTreeKey,
     disabled: false,
-    click: () => showSource(selectedSource.get("id"))
+    click: () => {
+      clearShownSource();
+      showSource(selectedSource.get("id"));
```

Comment:
ah although, we could listen to show source and clear it. thats also true -- i like your pattern of doing this on the reducer :+1: